nautobot/extras/health_checks.py

```diff
@@ -26,3 +27,45 @@ def check_status(self):
             raise ServiceReturnedUnexpectedResult("Integrity Error")
         except DatabaseError:
             raise ServiceUnavailable("Database error")
+
+
+class RedisBackend(BaseHealthCheckBackend):
+    """Health check for Redis."""
+
+    redis_url = getattr(settings, "CACHEOPS_REDIS", "redis://localhost:6379/1")
+    sentinel_url = getattr(settings, "CACHEOPS_SENTINEL", None)
+
+    def check_status(self):
+        """Check Redis service by pinging the redis instance with a redis connection."""
+        # if CACHEOPS_REDIS is set to False or Sentinel is enabled
+        # We need to check Redis using Sentinel
+        if not self.redis_url or self.sentinel_url is not None:
+
+            from redis.sentinel import Sentinel
```

Comment:
Is there a reason for the import at this location? If so, please add a comment.

Reply:
For ressource optimization. We do not need to import it if we don't use Sentinel.

Reply:
I moved the import at the beginning of the file because I've noticed, at each `/health` call, it call `check_status()` method ... My lazy import was useless